articles/mysql/single-server-whats-new.md

```diff
@@ -48,6 +48,10 @@ This release of Azure Database for MySQL - Single Server includes the following
 
   This release provides General Availability of planned maintenance notifications in Azure Database for MySQL - Single Server. For more information, see the article [Planned maintenance notification](concepts-planned-maintenance-notification.md).
 
+- **Known issue**
+
+  If you are using a PHP application connected to Azure database for MySQL with redirection enabled [Learn more]( https://docs.microsoft.com/en-us/azure/mysql/howto-redirection), you might face a data encoding issue which will impact your insert transactions , resolve the issue by disabling the redirection by setting the parameter redirect_enabled to “OFF” in Azure portal , or you can explicitly set the charset related parameters based on your settings at the session level after connection established , for example “set names utf8mb4”, and make sure to restart the PHP application to clear the driver cache after your changes.
```

Comment:
Please use this text instead:

If you are using a PHP application connected to Azure Database for MySQL with [redirection enabled](https://docs.microsoft.com/azure/mysql/howto-redirection), you might face a data encoding issue that will impact your insert transactions. To resolve the issue, in Azure portal, disable the redirection by setting the redirect_enabled parameter to “OFF”. You can also explicitly set the charset related parameters based on your settings at the session level after the connection is established, for example “set names utf8mb4”, and then restart the PHP application to clear the driver cache after your changes.